### ISSUE TITLE
Crash in other software accessing virtual cam

### ISSUE THREAD

**Issue by ttwang945** [obsproject/obs-studio#6598]
### Operating System Info

Windows 10

### OBS Studio Version

27.2.4

### OBS Studio Log URL

I have no log because this crash information comes from the user crash report

### Expected Behavior

prgram can operate stably

### Current Behavior

program crash

### Steps to Reproduce

I don't know because this crash information comes from the user crash report

### Anything else we should know?

thank for you notice this question.

User uses both obs and my software , stack is full of obs functions when my software crash.

I can receive this crash report more than 30 times a day.

dump stack image :
<img width="576" alt="dump_image" src="https://user-images.githubusercontent.com/50133565/172162041-b4caa570-b5f8-4498-a785-9dac0dfbbac1.png">

dump file:
[dump.zip](https://github.com/obsproject/obs-studio/files/8844491/dump.zip)

I would appreciate it if you can help me to solve the technical problem.

**Comment by ttwang945** [comment]
> Can you also tell us the name of your software and how you replicate the crash with it?

my software name is VooV Meeting (https://voovmeeting.com/).

I only get a dump file in crash report system and user not tell us how to trigger this crash. 

I tried to replicate this crash but it didn't work.

step for try:
1. run the OBS Studio.
2. click OBS Studio "Start Virtual Camera" button .
3. run the VooV Meeting and Log In.
4. click VooV Meeting "Start" button  for join a meeting.
5. click VooV Meeting bottom toolbar "Start Video" button after joined meeting.
6. Select OBS Virtual Camera where "Start Video" submenu.

**Comment by Fenrirthviti** [comment]
I've confirmed this in Discord, and it's easy to replicate there.

1. With the output in OBS inactive, go in to Settings -> Voice & Video in Discord
2. Select OBS Virtual Camera, and click "Test Video"
3. Start the virtual camera output in OBS
4. Switch to another settings tab in Discord (such as Accessibility)
5. Switch back to Voice & Video, and click Test Video again
6. Discord will crash

I don't have any other applications to test this in, but I would wager similar steps result in a similar crash. I am able to reproduce this 100% of the time, following those exact above steps.

**Comment by jp9000** [comment]
After some painstaking investigation, the crash @Fenrirthviti reported does not appear to be related to the original issue, and that one *appears* to be a bug with Discord itself, not the obs virtualcam filter. Happens regardless of scaling or OBS resolution, I checked the entire filter call list, and checked everything discord was sending to the virtualcam filter. It does not crash in the virtualcam code but instead in discord itself. I have no clue why that one is occurring, but I feel fairly confident in saying that whatever that one is, it does not appear to directly be the virtualcam's fault.

**Comment by notr1ch** [comment]
Some notes from analyzing the dump:

```cx: 1092
cy: 612

new_cx: 1092
new_cy: 612

OutputPin curCX: 1092
OutputPin curCY: 612

Scaler:
src_cx: 1092
src_cy: 614 <----- !!
dst_cx: 1920
dst_cy: 1080``

For whatever reason our scaler seems to be the wrong size.

**Comment by RytoEX** [comment]
Are you still receiving user crash reports on this?

**Comment by RytoEX** [comment]
We believe this crash was fixed in #7529, so we are closing this Issue. If you are still receiving crash reports, feel free to comment and we can consider reopening this or opening a new Issue.

**Comment by superconvert** [comment]
> We believe this crash was fixed in #7529, so we are closing this Issue. If you are still receiving crash reports, feel free to comment and we can consider reopening this or opening a new Issue.

Hello, RytoEx. this issue not fixed, I'm having a similar issue. I called OBS's virtual camera through the camera interface of webrtc, and I also got a crash.
OBS Version: 29.0.2 ( 64 bit ) or 30.0.2 ( 64 bit )
Video Size: 1092 x 614
I had patched notr1ch-vcam-resize-721e4fbe.zip and the issue persists
